views/js/qtiCommonRenderer/renderers/interactions/ExtendedTextInteraction.js

```diff
@@ -365,7 +365,7 @@ define([
         var $container = containerHelper.get(interaction);
         var attributes = interaction.getAttributes();
         var responseDeclaration = interaction.getResponseDeclaration();
-        var baseType = responseDeclaration.attr('baseType');
+        var baseType = responseDeclaration ? responseDeclaration.attr('baseType') : '';
```

Comment:
Agree, how `responseDeclaration` can be undefined at that point. May be the work made into https://github.com/oat-sa/extension-tao-itemqti/pull/366 (fix the ready state of interactions) would prevent that case to occur. 
The response cannot be retrieved without declaration.